### PR TITLE
Enable Chrome support by removing audio codec

### LINKED ISSUE #1: Does not work on Chrome
```
demo.js:38 Uncaught DOMException: Failed to execute 'appendBuffer' on 'SourceBuffer': This SourceBuffer has been removed from the parent media source.
    at WebSocket.ws.onmessage (http://192.168.1.42:2020/demo.js:38:20)
```

### PR DESCRIPTION
**Pull request by jwgcarlson**
Also touch up Makefile, README, and Go dependencies.

Fixes #1.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ enabled in `/boot/config.txt`.
 
 To fetch dependencies:
 
-    go get -v ./...
+    GOOS=linux go get -v ./...
```
Comment (hiszpanski) [inline README.md:24]: Is `GOOS` used by `go get`?

**Review by hiszpanski — APPROVED** [review]
Thank you for your contribution!

File: README.md (hunk 1 above)
Comment (jwgcarlson) [reply README.md:24]: Apparently. This is the error I get when I just run `go get -v ./...`:
```
$ go get -v ./...
github.com/thinkski/berrymse
# github.com/thinkski/berrymse
./main.go:153:10: undefined: v4l2.Device
./main.go:159:14: undefined: v4l2.Open
./main.go:168:3: undefined: v4l2.V4L2_PIX_FMT_H264
```

Comment (hiszpanski) [reply README.md:24]: Ah, good point -- v4l2 is linux-only.

**Pull request merged**